service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -1,827 +0,0 @@
-package org.mskcc.cmo.metadb.service.impl;
```

Comment:
Was this class intended to be deleted? It looks like there've been replacements for everything but this class still exists.

Reply:
I wanted to rename it to `RequestMessageHandlingServiceImpl`